eng/Versions.props

```diff
@@ -10,7 +10,7 @@
     <!--
         When StabilizePackageVersion is set to 'true', this branch will produce stable outputs for 'Shipping' packages
     -->
-    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
+    <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">true</StabilizePackageVersion>
```

Comment:
I think this should be false in master. @JunTaoLuo ?

Reply:
Yes it should stay as false. cc @mmitche

Reply:
Yep

Reply:
done.